UI/src/view/dynamicAssetForm.js

```diff
@@ -85,13 +85,13 @@
             return _.isUndefined(fieldSettings.showAndHide) ? true : fieldSettings.showAndHide(assetConfig.authorizationPolicy, assetConfig.selectedLinearAsset, value, layerMode) ? '' : element.hide();
         };
 
-        me.setSelectedValue = function(setValue, getValue){
+        me.setSelectedValue = function(setValue, getValue, silently) {
 
             var currentPropertyValue = me.hasValue() ?  me.getPropertyValue() : (me.hasDefaultValue() ? me.getPropertyDefaultValue() : me.emptyPropertyValue());
 
             var properties = _.filter(getValue() ? getValue().properties : getValue(), function(property){ return property.publicId !== currentPropertyValue.publicId; });
             var value = properties.concat(currentPropertyValue);
-            setValue({ properties: value});
+            setValue({ properties: value}, silently);
```

Comment:
This setValue can be the selectedLinearAsset.setValue() as you are seeing but it can be the setAValue or setBValue also so you will probably need to change those too, check please

Reply:
I will check, this is mainly used when affecting a property with some default value if this is not done the behaviour should stay the same

Reply:
setAValue and setBValue is used for creating two new linear assets and on those cases you want to trigger events so it won't be needed here